pyTigerGraph/pyTigerGraphBase.py

```diff
@@ -142,6 +126,34 @@ def __init__(self, host: str = "http://127.0.0.1", graphname: str = "MyGraph",
 
         self.Client = None
 
+        self.tgCloud = tgCloud or gcp
+        if "tgcloud" in self.netloc.lower():
+            try: # if get request succeeds, using TG Cloud instance provisioned before 6/20/2022
+                self._get(self.host + ":" + str(restppPort) + "/echo/" + graphname, resKey="message")
```

Comment:
Do we also need to set tgcloud = False here?

Reply:
No, because we set the `restppPort` and `restppURL` after this block. Line 132 only succeeds if there is no firewall config.